src/plugins/discover/public/application/main/utils/persist_saved_search.ts

```diff
@@ -8,12 +8,12 @@
 import { isOfAggregateQueryType } from '@kbn/es-query';
 import { DataView } from '@kbn/data-views-plugin/public';
 import { SavedObjectSaveOpts } from '@kbn/saved-objects-plugin/public';
+import { SavedSearch } from '@kbn/saved-search-plugin/public';
+import type { SortOrder } from '@kbn/saved-search-plugin/public';
```

Comment:
that's what this PR will do ... once the merge conflicts are resolved: https://github.com/elastic/kibana/pull/137488
Joe's PR went first, because I doubt he already has grey hair ... while one gray hair more for me, doesn't matter 👴